lib/protocol/collection/item_collection.ts

```diff
@@ -117,11 +117,25 @@ export class ItemCollection extends MutableCollection<SNItem> {
       /** Filtered content type map */
       const filteredCTMap = this.filteredMap[contentType]!;
       const sortedElements = this.sortedMap[contentType]!;
+
+      const currentIndex = filteredCTMap[element.uuid];
+      const currentElement = !isNullOrUndefined(currentIndex) ? sortedElements[currentIndex] : undefined;
+      if (currentElement?.errorDecrypting) {
+        if (element.errorDecrypting) {
+         /** if both elements are encrypted skip compare */
```

Comment:
Since they are the same elements (same UUID), this comment can be more helpful as:

```
/** If both previous and new values of element are errored, we simply replace the value, as a re-sort 
is not necessary, and the element will not be usable by forthcoming filters */
```

(Comments should use sentence case and so first letter should be capitalized)